.eslintrc.yml

```diff
@@ -0,0 +1,22 @@
+extends: wikimedia/server
```

Comment:
all other kartotherian projects extend 'kartotherian' for the rules, lets keep it consistent.